src/components/page/page.ts

```diff
@@ -138,17 +140,21 @@ export default class WaPage extends WebAwesomeElement {
   @property({ attribute: 'nav-open', reflect: true, type: Boolean }) navOpen = false;
 
   /**
-   * At what "px" to hide the "menu" slot and collapse into a hamburger button
+   * At what page width to hide the "menu" slot and collapse into a hamburger button.
+   * Accepts both numbers (interpreted as px) and CSS lengths (e.g. `50em`), which are resolved based on the root element.
```

Comment:
```suggestion
   * At what page width to hide the "navigation" slot and collapse into a hamburger button.
   * Accepts both numbers (interpreted as px) and CSS lengths (e.g. `50em`), which are resolved based on the root element.
```

Little nitpicky, but in talking with @lindsaym-fa it is a little confusing that the menu isn't technically what collapses, but the "navigation" slot that collapses. (turns from the `<nav>` to a `<dialog>`)